bin/init.sh

```diff
@@ -62,7 +66,7 @@ cd $INSTALL_DIR
 # create directory structure for docker volumes
 mkdir -p $INSTALL_DIR/data $INSTALL_DIR/data/minio $INSTALL_DIR/data/minio/bucket 
 mkdir -p $INSTALL_DIR/data/logs $INSTALL_DIR/data/analysis $INSTALL_DIR/tmp
-sudo mkdir -p /tmp/sagemaker
+mkdir -p /tmp/sagemaker
```

Comment:
My thinking was that if the folder was created as the current host user it would still be accessible if docker is running with sudo permission. I wasn't considering the folder already existing. Setting the group id to docker and adding group writable permission sounds like a good solution to me.

This logic is also duplicated in the init script should we look at removing https://github.com/larsll/deepracer-for-cloud/blob/master/scripts/training/start.sh#L53

Reply:
Docker Desktop for Mac runs as the host user and there is no docker group the default group id is wheel. So setting group writable should be sufficient.